spyder/plugins/ipythonconsole/confpage.py

```diff
@@ -117,8 +116,7 @@ def setup_page(self):
         if sys.platform.startswith('linux'):
             backends.append(("Gtk3", 5))
             backends.append(("Gtk", 6))
-        if PY2:
-            backends.append(("Wx", 7))
+        backends.append(("Wx", 7))
```

Comment:
There are wx-bindings for Python 3 now, so correct to change like this.

Reply:
Please remove this line instead of adding it back for Python 3. I plan to trim our list of supported backends in 5.3 to leave only the most used ones (i.e. Automatic, Qt5, Tkinter and macOS).